releases/sui/2025-01-ITS-v1.1.0.md

```diff
@@ -0,0 +1,152 @@
+# EVM ITS v1.1.0
+
+|  | **Owner** |
+|-----------|------------|
+| **Created By** | @milapsheth <milap@interoplabs.io>, @Foivos <foivos@interoplabs.io> |
+| **Deployment** | @blockchainguyy <ayush@interoplabs.io> |
+
+| **Network** | **Deployment Status** | **Date** |
+|-------------|----------------------|----------|
+| **Devnet Amplifier** | - | TBD |
+| **Stagenet** | - | TBD |
+| **Testnet** | - | TBD |
+| **Mainnet** | - | TBD |
+
+[Release](https://github.com/axelarnetwork/axelar-cgp-sui/releases/tag/v1.1.0)
+
+[Changelog](https://github.com/axelarnetwork/interchain-token-service/blob/v/CHANGELOG.md#210)
+
+## Deployment
+
+- Ensure that ITS Hub has been upgraded to [v1.1.0](../cosmwasm/2025-01-ITS-v1.1.0.md) first.
+- The rollout involves deploying Sui ITS contracts version `v1.1.0` on Sui.
+
+```bash
+# Clone latest main and update deps
+npm ci
+```
+
+Create an `.env` config. Use `all` for `CHAINS` to run the cmd for every EVM chain, or set a specific chain.
+
+```yaml
+MNEMONIC=<wasm-deployer-key>
+PRIVATE_KEY=<sui-deployer-key>
+PRIVATE_KEY_TYPE="mnemonic" # Optional
+SIGNATURE_SCHEME=secp256k1
+ENV=<devnet-amplifier|stagenet|testnet|mainnet>
+CHAIN=sui
+```
+
+### Deploy move contracts
+- Ensure that there isn’t a move directory in Sui directory, if it exists delete it.
+```bash
+rm -rf ./sui/move
+```
+
+- Deploy the ITS Contract and required depndencies on Sui (remember to save `sui/move` files for ITS deployment)
+
+```bash
+node sui/deploy-contract.js deploy Abi
+node sui/deploy-contract.js deploy InterchainTokenService
+node sui/deploy-contract.js deploy Example
+```
+
+### Register Sui chain on core
+
+```bash
+ITS_HUB_ADDR=$(cat ./axelar-chains-config/info/<your_config_name>.json | jq -rM '.axelar.contracts.InterchainTokenService.address')
+ITS_SUI_CHANNEL=$(cat ./axelar-chains-config/info/<your_config_name>.json | jq -rM '.chains.sui.contracts.InterchainTokenService.objects.ChannelId')
+```
+
+#### Devnet:
+```bash
+kubectl exec genesis-0 -- axelard tx wasm execute "$ITS_HUB_ADDR" \
+'{
+  "register_chains": {
+    "chains": [
+      {
+        "chain": "sui",
+        "its_edge_contract": "'"$ITS_SUI_CHANNEL"'",
+        "truncation": {
+          "max_decimals_when_truncating": '"8"',
+          "max_uint": "'"18446744073709551615"'"
+        }
+      }
+    ]
+  }
+}' \
+--from <name-of-your-wasm-deployer-key> \
+--gas auto \
+--gas-adjustment 1.2
+```
+
+#### Stagenet/Testnet:
+```bash
+node cosmwasm/submit-proposal.js its-hub-register-chains sui -t "Register ITS for Sui chain" -d "Register ITS for Sui chain at ITS Hub contract" --deposit <100000000 for stagenet and 2000000000 for testnet> \
```

Comment:
don't execute manually. Use the script instead with the config
https://github.com/axelarnetwork/axelar-contract-deployments/tree/main/cosmwasm#register-chain-on-its-hub-through-governance-proposal